lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/LanguageServerOptions.scala

```diff
@@ -21,5 +21,5 @@ case class LanguageServerOptions(
   secureRpcPort: Option[Int],
   dataPort: Int,
   secureDataPort: Option[Int],
-  jvmModeEnabled: Boolean
+  jvm: Option[Option[java.nio.file.Path]]
```

Comment:
dtto - what about `jvmModeEnabled: Boolean` and `jvmPath: Option[Path]`?

Reply:
I have to say I am very happy I haven't duplicated the variables after spending another hour on launcher: 8728e1ade0